packages/xo-server/src/xo-mixins/scheduling.js

```diff
@@ -77,7 +77,10 @@ export default class Scheduling {
         'schedules',
         () => db.get(),
         schedules =>
-          asyncMap(schedules, schedule => db.update(normalize(schedule))),
+          asyncMap(schedules, async schedule => {
+            await db.update(normalize(schedule))
+            this._start(schedule.id)
```

Comment:
I see that you start schedule after config import, but not disabling it after !
Maybe this PR should be named: fix start schedules after import